src/script/cryptography/CryptographyRepository.js

```diff
@@ -131,6 +77,11 @@ export class CryptographyRepository {
       const {userId, clientId} = ClientEntity.dismantleUserClientId(sessionId);
       amplify.publish(WebAppEvents.CLIENT.ADD, userId, {id: clientId}, true);
     });
+
+    if (useExistingCryptobox === true) {
```

Comment:
```suggestion
    if (useExistingCryptobox) {
```

Reply:
`useExistingCryptobox` should be a boolean, I don't want to wrongly accept numbers or characters.